3d_segmentation/torch/unet_inference_dict.py

```diff
@@ -65,12 +66,13 @@ def main(tempdir):
     dataloader = DataLoader(dataset, batch_size=2, num_workers=4)
     # define post transforms
     post_transforms = Compose([
+        Decollated(),
```

Comment:
`post_transforms` expects channel-first data, please use `decollate_batch` instead. `Decollated` is batch-first transform.